test/ShortestPaths/runtests.jl

```diff
@@ -0,0 +1,37 @@
+using SparseArrays
+using LightGraphs.ShortestPaths
+using LightGraphs.Traversals
+using LightGraphs.Traversals: NOOPSort
+
+import Base.==
+function ==(a::ShortestPaths.AStarResult, b::ShortestPaths.AStarResult)
+   return a.path == b.path && a.dist == b.dist
+end
```

Comment:
I am not sure if this a good idea, would prefer to have a new method name for testing such a thing, instead of implementing a new method for `==`, as this might change how the functions work.

Reply:
The problem is that we're returning a struct for A* now where before we were returning a vector of edges representing the path. Folks who are comparing outputs from two A*s will have their stuff break if we don't implement equality for the result.

Reply:
But this is in a test file.

Reply:
ah, you're right. We don't need that there. It's defined in `src`.